### PR TITLE
feat(forms container): add forms container

### PR DESCRIPTION
**Pull request by anler**
# PR Prelude

Thank you for contributing to sheikah! :)

**Please fill in this template (by putting an `x` inside
the brackets) _before_ filing your PR:**

- [X] I have read and understood [CODE_OF_CONDUCT][code] document.
- [X] I have read and understood [CONTRIBUTING][cont] document.
- [X] I have read and understood [STYLEGUIDE][style] document.
- [X] I have included tests for the changes in my PR. If not, I have included a
  rationale for why I haven't.
- [X] My code is formatted and is accepted by `yarn fmt`.
- [X] **I understand my PR may be closed if it becomes obvious I didn't
  actually perform all of these steps.**

# Why this change is necessary and useful

[Please explain **in detail** why the changes in this PR are needed.]
Add a forms container. This PR only creates the container with the routes, fill the container with the common stuff will be done in another task.

Fixes #318 #301 

[code]: https://github.com/witnet/sheikah/blob/master/.github/CODE_OF_CONDUCT.md
[cont]: https://github.com/witnet/sheikah/blob/master/.github/CONTRIBUTING.md
[style]: https://github.com/witnet/sheikah/blob/master/docs/STYLEGUIDE.md

### CODE REVIEW

File: app/renderer/routes.tsx [hunk 1]
```diff
@@ -28,19 +24,14 @@ export default (props: RoutesProps) => {
...
-          render={loginRenderer}
+          path="/forms"
+          component={Forms}
         />
         <RedirectedRoute
           exact={true}
           path="/"
           guard={ifWallets(props.store)}
-          locationA={LOGIN_FORM_PATHS.WALLET_SELECTION}
+          locationA="/forms/login"
```
Comment (aesedepece) [inline app/renderer/routes.tsx]: Let's centralize and export all path literals in a separate file :wink:

File: package.json [hunk 2]
```diff
@@ -39,8 +39,9 @@
     "precommit": "lint-staged",
     "commitmsg": "commitlint -g config/commitlint.config.js -e $GIT_PARAMS",
     "postinstall": "electron-builder install-app-deps",
-    "fmt": "tsfmt --useTsfmt config/tsfmt.json --verify",
-    "fmt!": "tsfmt --useTsfmt config/tsfmt.json -r",
+    "_fmt":"tsfmt --useTsfmt config/tsfmt.json --useTsconfig config/tsconfig.fmt.json",
+    "fmt": "yarn _fmt --verify",
+    "fmt!": "yarn _fmt -r",
```
Comment (aesedepece) [inline package.json]: Please refer issue #301 in the comment of this commit.

File: app/common/constants/urls.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const forms = "/forms"
+export const login = "/forms/login"
+export const newWallet = "/forms/wallet"
```
Comment (mmartinbar) [inline app/common/constants/urls.ts]: IMO, this file should not be placed in common folder but in renderer instead given that the main process won't be using these urls

File: app/common/constants/urls.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+export const FORMS = "/forms"
+export const LOGIN = "/forms/login"
+export const NEW_WALLET = "/forms/wallet"
+
+export const MAIN = "/main"
```
Comment (mmartinbar) [inline app/common/constants/urls.ts]: IMO, this file should not be in common folder but in renderer, given that these urls won't be used by main process

**Comment by aesedepece** [comment]
:+1:, great point

>

File: app/common/constants/urls.ts (hunk 4 above)
Comment (anler) [reply app/common/constants/urls.ts]: I thought they could come handy if for some reason the backend needed to tell the frontend: hey, go to this url, but yeah I think is better to leave it in renderer and solve that problem in  the future if it's really needed :dancer:

**Pull request merged**